### PR TITLE
[Dashboard] Feature: Show subscriptions in Billing Page

### PR DESCRIPTION
**Pull request by jnsdls**
## PR-Codex overview
This PR focuses on updating the billing and subscription management features in the dashboard application. It refines routing, adds new API integrations, and modifies components to handle team subscriptions more effectively.

### Detailed summary
- Updated URLs for subscription plans to include `/billing`.
- Introduced `getStripeBillingPortalLink` API function for billing portal access.
- Modified `Billing` component to utilize subscriptions instead of account usage.
- Refactored `PlanInfoCard` to display subscription details.
- Added new `teamSubscriptionsStub` for testing.
- Enhanced error handling for subscription fetching.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by jnsdls** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5447?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#5447** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5447?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5447?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* **#5432** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5432?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 1 other dependent PR ([#5436](https://github.com/thirdweb-dev/js/pull/5436) <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5436?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#5430** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5430?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

**Review by gregfromstl — APPROVED** [review]

File: apps/dashboard/src/@/api/team-subscriptiion.ts [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    case "usage:in_app_wallet":
+      return "In-App Wallet";
+    case "usage:aa_sponsorship":
+      return "AA Sponsorship";
+    case "usage:aa_sponsorship_op_grant":
+      return "AA Sponsorship Op Grant";
+    default:
+      return null;
+  }
+}
```
[the remark this answers is not in the document]
Comment (MananTank) [reply apps/dashboard/src/@/api/team-subscriptiion.ts]: fixed

File: apps/dashboard/src/@/api/team-subscriptiion.ts [hunk 2]
```diff
@@ -0,0 +1,106 @@
+import { getAuthToken } from "../../app/api/lib/getAuthToken";
```
[the remark this answers is not in the document]
Comment (MananTank) [reply apps/dashboard/src/@/api/team-subscriptiion.ts]: fixed

File: apps/dashboard/src/@/api/team-subscriptiion.ts [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    case "usage:in_app_wallet":
+      return "In-App Wallet";
+    case "usage:aa_sponsorship":
+      return "AA Sponsorship";
+    case "usage:aa_sponsorship_op_grant":
+      return "AA Sponsorship Op Grant";
+    default:
+      return null;
+  }
+}
```
[the remark this answers is not in the document]
Comment (MananTank) [reply apps/dashboard/src/@/api/team-subscriptiion.ts]: fixed

**Pull request merged**